src/Lib/ErrorQueryExecutor.php

```diff
@@ -13,58 +13,54 @@
 
 use Exception;
 use Manticoresearch\Buddy\Enum\ManticoreEndpoint;
-use Manticoresearch\Buddy\Interface\BuddyLocatorClientInterface;
-use Manticoresearch\Buddy\Interface\BuddyLocatorInterface;
 use Manticoresearch\Buddy\Interface\CommandExecutorInterface;
 use Manticoresearch\Buddy\Interface\ErrorQueryRequestInterface;
 use Manticoresearch\Buddy\Interface\ManticoreHTTPClientInterface;
-use Manticoresearch\Buddy\Interface\ManticoreResponseBuilderInterface;
+//@codingStandardsIgnoreStart
```

Comment:
We should remove comments like this due to it's fixed already in the ruleset.xml, and PHPCS will not remove them when they are used only in comment blocks for type hinting